### DISCUSSION TITLE
Reading the Output

### DISCUSSION THREAD

**Discussion by varunkr58** [gorules/zen-go#25] [Q&A] [answered]
I tried executing a basic program to test out zen, but the output is all numbers, do I have to parse or convert this to any other format to read?
Program:
![Screenshot from 2024-07-17 08-01-47](https://github.com/user-attachments/assets/115ead7d-51d4-4416-95c2-9a1aaa3d0342)

Output :
`&{85.073µs [123 34 111 117 116 112 117 116 34 58 34 69 81 85 65 76 32 84 79 32 49 53 34 125] <nil>}`

**Comment by stefan-gorules** [comment]
Hi @varunkr58

Could you provide a code example (repository). Please try to update zen-go to `v0.5.1` as we've had an issue with `v0.5.0`.

**Reply by varunkr58** [reply, marked as answer]
Thanks for reply @stefan-gorules , it was just converting json Raw Message to String, I changed the print line in my program from `fmt.Println(output)` to `fmt.Println(string(output.Result))`.
